smart-py/marketplace.py

```diff
@@ -6,8 +9,11 @@ def __init__(self, objkt, metadata, manager, fee):
             manager = manager,
             swaps = sp.big_map(tkey=sp.TNat, tvalue=sp.TRecord(issuer=sp.TAddress, objkt_amount=sp.TNat, objkt_id=sp.TNat, xtz_per_objkt=sp.TMutez, royalties=sp.TNat, creator=sp.TAddress)),
             counter = 500000,
-            fee = fee
-            )
+            fee = fee,
+            fee_account = manager,
```

Comment:
It might be good to understand the roles and responsibilities of each of the kinds of managers.
Also, what conditions and processes would be followed to change any of these.

Reply:
The manager has control of everything provided he pings regularly. I would recommend that he changes the fee_account where the h=n fee goes regularly, to avoid problems of missing the keys etc. 
The problem with the bus accident, is that even if the admin account can be changed, the money in the last fee_account will be lost, since the new manager will not have the keys for that wallet. However, the new manager can change the fee_account to a new account of him.

Reply:
So the normal procedure of the manager will be:
- ping the contract regularly
- change the fee_account when it has too much funds. Maybe move the funds from the previous fee_account to a wallet controlled by the person delegated to pay dev, infra, etc.
- Keep the the secondary_accounts updated with users he trust and could take control in case of a "bus accident"

Reply:
Another option is that the fees stay in the marketplace contract balance. In that case the money will not be lost in case of bus accident. 
We can add another entrypoint called transfer_funds(address) that can only executed by the manager to transfer the funds to another wallet.